src/ranking_collecter/ranking_collector.py

```diff
@@ -0,0 +1,26 @@
+from string import Template
+import requests
+import user_info
+
+def collect_ranks(event, context):
+    usernames = user_info.USERNAMES
+    weekly_rankings = []
+    contest = user_info.CONTEST
+    weeklyContest = user_info.WEEKLYCONTEST
```

Comment:
add an update me comment to change this to use an endpoint from core-service to collect this data